src/back/spv/writer.rs

```diff
@@ -1033,6 +1033,76 @@ impl Writer {
                 }
                 _ => unimplemented!("{:?}", origin),
             },
+            crate::Expression::As {
+                expr,
+                kind,
+                convert,
+            } => {
+                // TODO Convert = false, a bitcast, is not yet implemented in the front-ends
```

Comment:
it would actually be relatively simple to add, there is a single `OpBitcast`

Reply:
Added it, but I didn't test it, as we don't pass `convert = true` from the front-ends yet.